### ISSUE TITLE
SIGBUS in gx_engine::NeuralAmp::compute

### ISSUE THREAD

**Issue by edwloef** [brummer10/guitarix#190]
This occurs when changing the buffer size at samplerates other than 48000, with the NAM plugin loaded. Can't tell whether this is another pipewire-jack bug though, since the stacktrace isn't very helpful:  [bt.txt](https://github.com/user-attachments/files/17533364/bt.txt)

**Comment by edwloef** [comment]
gdb claims this to be the line it crashes at, which doesn't make much sense to me either: https://github.com/brummer10/guitarix/blob/be6310d565df7d34bb06ba08a69c44393c91c7d4/trunk/src/gx_head/engine/gx_internal_plugins.cpp#L1923

**Comment by edwloef** [comment]
In the git version, it crashes somewhere in here: https://github.com/brummer10/guitarix/blob/9013250487851fb98bb936c9ca5d210cfb560ef0/trunk/src/gx_head/engine/gx_internal_plugins.cpp#L1925-L1927

**Comment by brummer10** [comment]
Does that mean that it didn't crash under pipewire when you change the buffersize any more, without a neural model is running?

**Comment by edwloef** [comment]
Yes, the old crash no longer occurs since pipewire-jack 1.2.6, this commit fixed it: https://gitlab.freedesktop.org/pipewire/pipewire/-/commit/27961630a669f0a58ccbfe044cd0c3ebecbe77c0

**Comment by brummer10** [comment]
Ah, that's nice to hear. 
Well, I just did some quick tests under jack and can't reproduce the crash under jack. 
As I'm still on pipewire 1.2.0 I can't test on pipewire, but, as it works flawless under jack I suspect pipewire to be the culprit.
